gradle-plugin/src/main/kotlin/com/google/devtools/ksp/gradle/KspSubplugin.kt

```diff
@@ -514,6 +514,7 @@ class KspGradleSubplugin @Inject internal constructor(private val registry: Tool
                         kspTask.compilerOptions.freeCompilerArgs.value(
                             kspOptions + kotlinCompileTask.compilerOptions.freeCompilerArgs.get()
                         )
+                        kspTask.compilerOptions.useK2.value(false)
```

Comment:
is this related to this fix?

Reply:
Not specific to #1280, but a fix that is small enough to come with other commits.